### PR TITLE
Delete a useless auto generated directory.

### PR DESCRIPTION
**Pull request by keisuke-umezawa**
## Motivation

To solve the issue https://github.com/optuna/optuna/pull/1400#discussion_r473617649

## Description of the changes

Use the `genereted` directory for all auto summary documenation.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -112,6 +112,5 @@ ENV/
 tutorial/example.db
 docs/_build/
 docs/source/reference/generated/
-docs/source/reference/alias_generated/
```
Comment (toshihikoyanase) [inline .gitignore:115]: Maybe we can remove this directory from the target of `make clean`.

Comment (keisuke-umezawa) [reply .gitignore:115]: Thank you! I fixed it.

**Review by hvy — APPROVED** [review]
LGTM!

**Review by toshihikoyanase — APPROVED** [review]
Thank you for your update. LGTM!

**Pull request merged**